data/migrations/20211109194115__delete-old-stories-table.js

```diff
@@ -0,0 +1,14 @@
+exports.up = function (knex) {
+  return knex.schema
+    .alterTable('Submissions', (t) => {
+      t.dropForeign('StoryID');
+    })
+    .alterTable('Cohorts', (t) => {
+      t.dropForeign('StoryID');
+    })
+    .dropTableIfExists('Stories', () => {});
+};
+
+exports.down = function () {
+  // This is a destructive migration that cannot be rolled back.
+};
```

Comment:
I suggest adding this to the down migration so that the table shape is restored
```js
exports.down = function (knex) {
  return knex.schema
    .createTable('Stories', (t) => {
      t.increments('ID');
      t.string('Title').notNullable();
      t.string('URL').notNullable();
      t.string('WritingPrompt').notNullable();
      t.string('DrawingPrompt').notNullable();
      t.string('Audiofile').nullable();
    })
    .alterTable('Submissions', (t) => {
      t.foreign('StoryID').references('Stories.ID');
    })
    .alterTable('Cohorts', (t) => {
      t.foreign('StoryID').references('Stories.ID');
    })
};
```

Reply:
Thanks for the immediate review Chris. The changes you requested have been added. Please [refer](https://github.com/BloomTech-Labs/scribble-stadium-be/pull/96/commits/8a293e1e40df9dbffb64b2bdb82c0ec64c73b855)